docs/how-to/FAQ.md

```diff
@@ -4,4 +4,9 @@
 
 If you see the error below. You have to ensure that the service connection is created at the Azure Machine Learning Workspace level and not Subscription level
 
-![faq-error-invoke](../media/FAQ-invoke.error.png)
\ No newline at end of file
+![faq-error-invoke](../media/FAQ-invoke.error.png)
+
+## I would like to adjust the Docker Image. How can I change the Dockerfile?
+
+Documentation can be found here: 
+**[Custom Dockerfile](dstoolkit-mlops-base/docs/how-to/AdjustingDockerfile.md)**
```

Comment:
The link doesn't work (check [here](https://github.com/microsoft/dstoolkit-mlops-base/blob/4d450dcce572c6f62ade8097826e33a892b29e8d/docs/how-to/FAQ.md)).